litr/src/main/java/com/linkedin/android/litr/resample/AudioResampler.java

```diff
@@ -0,0 +1,28 @@
+package com.linkedin.android.litr.resample;
+
+import androidx.annotation.NonNull;
+import java.nio.ByteBuffer;
+
+/**
+ * Resamples audio data. See {@link UpsampleAudioResampler} or
+ * {@link DownsampleAudioResampler} for concrete implementations.
+ */
+public interface AudioResampler {
```

Comment:
For all new classes that are not modified classes from other project please add LinkedIn copyright notice